planner/optimizer/src/main/java/eu/seaclouds/platform/planner/optimizer/CloudOffer.java

```diff
@@ -91,7 +97,8 @@ public void setCost(double cost) {
 
    @Override
    public CloudOffer clone() {
```

Comment:
It was an opinion, if it's working no problem with this. I just wanted to notice you.